articles/overview/how-to-learn-quantum-computing.md

```diff
@@ -0,0 +1,46 @@
+---
+title: How to learn quantum computing
+description: 
+author: natke
+ms.author:  nakersha
+ms.date: 10/16/2019
+ms.topic: article
+uid: microsoft.quantum.overview.learn
+---
+
+# How to learn quantum computing
+
+Learn how to write your first quantum program.
+
+## Getting the basics: what do you need to know?
+
+Fortunately, you don’t need to know quantum physics to start writing quantum applications. However, there are some concepts that are especially important to understand quantum computing. But don’t worry if you don’t know something! We will give you the resources to learn the necessary tools in a light and interactive way.
+
+* Complex numbers: quantum mechanics makes use of complex numbers to express quantum states. They are crucial to understand the basics of quantum computation. Here you can find a tutorial to get everything you need to know about complex numbers.
+* Linear algebra (vector and matrices): quantum states are vectors in a vector space over the complex numbers. Quantum gates are just linear operations acting on these states. Here you can find a tutorial to get everything you need to know about linear algebra. (Link to a Binder for the tutorial on linear algebra).
+* Basic quantum mechanics: ok, I know we just said that you don’t need to know quantum physics to start coding (and it’s true!). But some basic concepts of quantum mechanics and its mathematical notation will be helpful to understand quantum programming. Here you can find a tutorial to get everything you need to know about quantum mechanics. (Link to a Binder for the tutorial on basic quantum mechanics).
+
+## Code first, then learn!
+
+Now that we have the basics, we are ready to start writing our first quantum programs. There are many options to get started. Here we recommend you some of them:
+
+* [Write your first quantum program](xref:microsoft.quantum.write-program)
+* Learn with the [Quantum Katas](https://github.com/microsoft/QuantumKatas): an open-source project containing programming exercises aimed at teaching quantum computing. In particular, the online version of the Quantum Katas in form of Q# Jupyter Notebooks is especially easy to use. In less than 5 min you can start to write your first quantum program without needing to install anything.
+* [Tutorials and samples]()
```

Comment:
Do you want this to link to the samples repo or the samples browser (once we get everything on-boarded OK)?

Reply:
Bumping this, now that said samples are onboarded at https://docs.microsoft.com/samples/browse/?languages=qsharp